README.md

```diff
@@ -2147,7 +2186,7 @@ You can restrict icons to only show for certain [views](#views) using a
 `custom:frigate-card-conditional` element (e.g. PTZ controls)
 
 <details>
-  <summary>Expand: View-based conditions (e.g. PTZ controls)</summary>
+  <summary>Expand: View-based conditions (e.g. custom PTZ controls)</summary>
 
 This example shows PTZ icons that call a PTZ service, but only in the `live` view.
```

Comment:
I think this section could be deleted entirely after introducing the new ptz element.

Reply:
I'll just change the example to non-PTZ (the intent of the example is more about conditions).